Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorInteractionSystemComponent.cpp

```diff
@@ -56,6 +56,12 @@ namespace AzToolsFramework
         return m_interactionRequests->InternalHandleMouseManipulatorInteraction(mouseInteraction);
     }
 
+    const EditorVisibleEntityDataCacheInterface* EditorInteractionSystemComponent::GetEntityDataCache() const
+    {
+        return m_entityDataCache.get();
+    }
+
+
```

Comment:
uber nit: Additional line break, this class just usually has one between member functions in the .cpp